documentation/release_notes.markdown

```diff
@@ -7,6 +7,49 @@ canonical: "/puppetdb/latest/release_notes.html"
 [configure_postgres]: ./configure.html#using-postgresql
 [pg_trgm]: http://www.postgresql.org/docs/current/static/pgtrgm.html
 
+2.3.2
+-----
+
+PuppetDB 2.3.2 is a backwards-compatible bugfix release that corrects
+two problems which could prevent migration to the database format
+introduced in 2.3.1.
+
+### Upgrading
+
+* For the best-possible performance and scaling capacity, we recommend
+  PostgreSQL version 9.4 or newer with the [`pg_trgm`][pg_trgm]
+  extension enabled, as explained [here][configure_postgres], and we
+  have officially deprecated versions earlier than 9.2.  HSQLDB is
+  only recommended for local development because it has a number of
+  scaling and operational issues.
+
+* Make sure that all of your PuppetDB instances are shut down, and
+  only upgrade one at a time.
+
+* Make sure to upgrade your puppetdb-terminus package (on the host
+  where your Puppet master lives), and restart your master service.
+
+### Contributors
+
+Rob Browning
+
+#### Bug Fixes and Maintenance
+
+* The database migration required by the garbage collection fix
+  in 2.3.1 should no longer refuse to complete when the existing
+  database contains different values with the same path in different
```

Comment:
@wkalt oh, right, the type has to be different in order to create multiple paths.